DeepNtuplizer/python/ConfFile_cfg.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Can you move this file to test/? Generally cfg files should not enter in python/.

Reply:
okay